packages/notifi-frontend-client/lib/models/SubscriptionCardConfig.ts

```diff
@@ -75,6 +75,19 @@ export type TradingPairEventTypeItem = Readonly<{
   tradingPairs: ValueOrRef<ReadonlyArray<string>>;
 }>;
 
+export type FusionMultiThreshholdEventTypeItem = Readonly<{
+  type: 'fusionMultiThreshhold';
+  name: string;
+  tooltipContent?: string;
+  numberType: NumberTypeSelect;
+  subtitle?: string;
+  addThreshholdTitle?: string;
+  fusionEventId: ValueOrRef<string>;
+  sourceAddress: ValueOrRef<string>;
+  maintainSourceGroup?: boolean;
+  alertFrequency?: AlertFrequency;
+}>;
+
```

Comment:
Since it is also under `FusionEventTypeItem`, so I might suggest to re-use `FusionTypeBase` and use the `selectedUIType` to distinguish this one from others. like below
```
export type FusionEventTypeItem =
  | FusionToggleEventTypeItem
  | FusionHealthCheckEventTypeItem
  | FusionMultiThresholdEventTypeItem

 export type FusionMultiThresholdTypeItem = FusionTypeBase &
  Readonly<{
    selectedUIType: 'MULTI_THRESHOLD';
    // ....
  }>;
```

Reply:
hm based on my comment here https://github.com/notifi-network/notifi-sdk-ts/pull/382#discussion_r1323394058, I think fusionMultiThreshhold should not be a subtype of fusion. 

I can use fusiontypebase with omits to reduce redundancy though

Reply:
i think if fusionMultithreshold is a subtype of fusion, we will run into issues where we will see multiple toggles for the same multithreshold alert when the user signs up for multiple thresholds

Reply:
Not sure what does the `filterOption` of this alert look like. 
I though it might be single fusion source. And then BE uses it's `filterOtpion` to decide what to emit . If it is the case, it would not have multiple toggles (since the each threshold is defined in filterOption)